Brokerages/DefaultOrderBook.cs

```diff
@@ -89,9 +97,9 @@ public void Clear()
         /// <param name="size">The new size at the bid price level</param>
         public void UpdateBidRow(decimal price, decimal size)
         {
-            lock(_locker)
+            lock (_locker)
             {
-                _bids[price] = size;
+                Bids[price] = size;
             }
 
             if (BestBidPrice == 0 || price >= BestBidPrice)
```

Comment:
Since we should synchronize access to all `Bestxxx` properties, we should also move this `if` block inside the `lock` block above -- same for `UpdateAskRow`